inventory-framework-platform-bukkit/src/main/java/me/devnatan/inventoryframework/runtime/thirdparty/ReflectionUtils.java

```diff
@@ -182,9 +153,9 @@ public static String getVersionInformation() {
     private static final MethodHandle SEND_PACKET;
 
     static {
-        Class<?> entityPlayer = getNMSClass("server.level", "EntityPlayer");
-        Class<?> craftPlayer = getCraftClass("entity.CraftPlayer");
-        Class<?> playerConnection = getNMSClass("server.network", "PlayerConnection");
+        Class<?> entityPlayer = Objects.requireNonNull(getNMSClass("server.level", "EntityPlayer"));
```

Comment:
Well, Objects.requireNonNull is still not quite needed here.

Existing code should remain unchanged if not necessary to Implement a feature